.github/workflows/bundler.yml

```diff
@@ -28,17 +31,17 @@ jobs:
           branch: ${{ github.ref }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: pull-request
-        uses: romaninsh/pull-request@master
-        with:
-          source_branch: "release/${{ env.version }}"
-          destination_branch: "master"          # If blank, default: master
-          pr_title: "Releasing ${{ env.version }} into master"
-          pr_body: |
-            - [ ] Review changes (must include stable dependencies)
-            - [ ] Merge this PR into master (will delete ${{ github.ref }})
-            - [ ] Go to Releases and create TAG from master
-            Do not merge master into develop
-          pr_reviewer: "romaninsh"
-          pr_assignee: "romaninsh"
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+      #- name: pull-request
```

Comment:
definitely do not comment out old code, simply remove the code

Reply:
yes i will cleanup/remove at the end before merging